### PR TITLE
t/ckeditor5-ui-default/12: Implement dropdown component

### PR DESCRIPTION
**Pull request by oleq**
Related to https://github.com/ckeditor/ckeditor5-ui-default/issues/12.

### CODE REVIEW

File: src/utils/imitatefeatures.js [hunk 1]
```diff
@@ -37,12 +43,15 @@ export function imitateFeatures( editor ) {
...
+	// -------------------------------------------------------------------------------------------
+
 	/* istanbul ignore next */
 	const italicModel = new Model( {
 		isEnabled: true,
 		isOn: false,
 		label: t( 'Italic' ),
-		icon: 'italic'
+		icon: 'italic',
+		iconAlign: 'left'
```
Comment (Reinmar) [inline src/utils/imitatefeatures.js]: Should be uppercase `'LEFT'`. We define elements of dictionaries using `'UPPER_CASE'` strings.

File: tests/manual/classiccreator.js [hunk 2]
```diff
@@ -45,3 +45,5 @@ function destroyEditor() {
 
 document.getElementById( 'initEditor' ).addEventListener( 'click', initEditor );
 document.getElementById( 'destroyEditor' ).addEventListener( 'click', destroyEditor );
+
+initEditor();
```
Comment (Reinmar) [inline tests/manual/classiccreator.js]: The test description is now incorrect. Change the description or remove that line.

**Comment by Reinmar** [comment]
After a bit of thinking I'm afraid that this entire PR needs to be rejected. We should not test the dropdown inside the classic creator. Those tests should be added to the ckeditor5-ui-default repo.

File: tests/manual/classiccreator.js (hunk 2 above)
Comment (oleq) [inline tests/manual/classiccreator.js]: Yep. The line was used in dev env only. Forgot to remove it.

**Comment by oleq** [comment]
So why testing buttons and not testing dropdowns in creator test?

**Comment by Reinmar** [comment]
We don't test buttons here. We test the creator with some initial content. That features mock will soon be replaced with real features, but again, only a few, and the test should not change every time we have something new to test. This is creator's test.

**Comment by oleq** [comment]
IMO dropdowns are essential for the editor. They should stay in this test but yes, there's no point in extending it any further.

**Comment by Reinmar** [comment]
That's make a bit of a sense, taken the usage of certain components like the floating toolbar. So ok, let's keep those here, they will later be replaced with basic styles + formats.

So the thing that's missing is a manual test in the UI-default repo.

**Comment by oleq** [comment]
> So the thing that's missing is a manual test in the UI-default repo.

AFAIR I created MT for dropdown component itself: https://github.com/ckeditor/ckeditor5-ui-default/pull/15/files#diff-a7019e1df44cbb48db4a721a38ec05fbR1

**Comment by Reinmar** [comment]
> AFAIR I created MT for dropdown component itself: https://github.com/ckeditor/ckeditor5-ui-default/pull/15/files#diff-a7019e1df44cbb48db4a721a38ec05fbR1

LOL. I looked for it and missed it :D Sorry.

**Pull request merged**